userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -105,10 +105,12 @@ _Example:_
 
 |===
 
-External Payment Method:: A type of payment method that enables you to track payments that occur outside of Kill Bill (_Example:_ check payment). Its functionality is provided as part of the Kill Bill payment system. 
+External Payment Method:: A type of payment method that enables you to track payments that occur outside of Kill Bill (_Example:_ check payment). Its functionality is provided as part of the Kill Bill payment system.
 
 == F
 
+Fixed Offset Timezone:: A special timezone used by the system for dates manipulation. It is used to determine the date in the user's time zone.
```

Comment:
Is this referring to the `account#time_zone` - In which case should we explain this can be set at the `Account` (customer) level, and explain how the system uses it?  Can you point to an example below to follow?

Reply:
I'll point to an example once we finalize the examples in the other document. 👍